### DISCUSSION TITLE
Voti scrutinio per tipologia

### DISCUSSION THREAD

**Discussion by mimmomarino** [iisgiua/giuaschool#797] [Idee e proposte]
Buongiorno,
In quanto il nostro istituto adotta la suddivisione delle valutazioni in scritto, orale e pratico, sarebbe opportuno se durante le fasi di scrutinio del primo quadrimestre si potesse suddividere le valutazioni in questo modo, anzicché inserire il voto unico.
Grazie in anticipo

**Comment by trinko** [comment]
Questo richiede molte modifiche nella gestione degli scrutini.
Attualmente non è previsto.

**Comment by scarfone-d** [comment]
Buongiorno,
è possibile in alternativa inserire un campo aggiuntivo come giudizio per materia ??? di modo da specificare che il voto è una sintesi tra differenti tipologie - as esempio scritto / orale - grazie
